pkg/core/runtime/reports/reports.go

```diff
@@ -69,6 +71,35 @@ func fetchZones(rt core_runtime.Runtime) (*system.ZoneResourceList, error) {
 	return &zones, nil
 }
 
+func fetchNumPolicies(rt core_runtime.Runtime) (map[string]string, error) {
+	policyTypes := map[string]core_model.ResourceList{
+		"health_checks":       &mesh.HealthCheckResourceList{},
+		"traffic_traces":      &mesh.TrafficTraceResourceList{},
+		"traffic_routes":      &mesh.TrafficRouteResourceList{},
+		"retries":             &mesh.RetryResourceList{},
+		"traffic_permissions": &mesh.TrafficPermissionResourceList{},
+		"traffic_logs":        &mesh.TrafficLogResourceList{},
+		"fault_injections":    &mesh.FaultInjectionResourceList{},
+		"timeouts":            &mesh.TimeoutResourceList{},
+		"rate_limits":         &mesh.RateLimitResourceList{},
+		"circuit_breakers":    &mesh.CircuitBreakerResourceList{},
+		"zone_ingresses":      &mesh.ZoneIngressResourceList{},
+		"zone_egresses":       &mesh.ZoneEgressResourceList{},
+		"gateway_routes":      &mesh.MeshGatewayRouteResourceList{},
+		"proxy_templates":     &mesh.ProxyTemplateResourceList{},
+	}
```

Comment:
I think we could do better here. What about we go through our registry of resources? This way we don't have to think about adding stuff there when we add a new resource.
The only thing is that the metric name will change from `rate_limits` to `RateLimit` etc.

Reply:
should be ok imho, but please check with @subnetmarco

If not, we can convert camel case to snake case

Reply:
Marco suggested converting to lowercase, so will do that.